### ISSUE TITLE
ubuntu server iso not booting

### ISSUE THREAD

**Issue by mremptypants** [netbootxyz/netboot.xyz#1436]
It gets to pulling the iso from release.ubuntu.com and than fails to pull and mount, I have tried with docker, with, without custom config. It pulls the other rls so dns and conifg seems good just not ubuntu for whatever reason. Not a big deal being i do not use it much.  

The vlan boots, working installs on all os i have tried..if it is user error please point me in right area. 

<img width="1037" alt="Screenshot 2024-03-22 at 11 46 04 AM" src="https://github.com/netbootxyz/netboot.xyz/assets/162651217/93bc7aef-aaa2-4f09-8010-6ce2314c81c5">

**Comment by ricocarlsson** [comment]
can confirm this isnt working.
![image](https://github.com/user-attachments/assets/cecca2b9-2d3e-4740-87b3-8bb9935df6a9)

seems they changed / removed the file name / version
should now be https://releases.ubuntu.com/noble/ubuntu-24.04.1-live-server-amd64.iso

they basically added a .1 in the file name.

**Comment by chris18890** [comment]
Indeed, the point release dropped - https://lists.ubuntu.com/archives/ubuntu-announce/2024-August/000304.html

**Comment by cryptonym64** [comment]
Fails now on 24.04.2   It appears they are now on .3

**Comment by cryptonym64** [comment]
It appears a design bug in the template(s). I found that if instead of going to https://releases.ubuntu.com/noble/  the files are really at: http://releases.ubuntu.com/releases/24.04.2  (yes, x.3 is also same location).

Basically, ignore the codename and use simple version numbers for the path.  Codename may help for searching, but it's throwing a wrench in on going straight to the files needed.

Hope this helps a dev sort this problem. I just am not sure what to change or where.

**Comment by cryptonym64** [comment]
I just hand edited my local ubuntu.ipxe. Changed .2 to .3 and left what looked to be a hash on it. It worked!  I then paused during a boot and got the url, pointing into github. I went to the files area and surprisingly the .3 file was there, with the code on the end. So, somehow it is getting build in github repo, but not making it into the menus it seems.  I'm running 2.0.87. Now I see there is a 2.0.88, will try that later.

**Comment by cryptonym64** [comment]
newest menus has the .3 already in it. Maybe the squash repo needs to retain copies of the old files too???
